yaml/boundaries.yaml

```diff
@@ -276,6 +279,7 @@ filters:
       <<: [*output_properties, *ne_localized_kind_properties]
       kind: *ne_country_boundaries_kind
       kind_detail: '2'
+      dispute_id: { col: 'ne:brk_a3' }
```

Comment:
To indicate both the dispute grouping and the unique line segments, concat with `ne_id` with a `_` spacer. The result would be something like `B05_123456789`

Reply:
(Same comment as above, repeated for this separate config.)

Reply:
And repeats below...

Reply:
Switch to using global, please.